src/app/(main)/layout.tsx

```diff
@@ -1,29 +1,19 @@
-// @ts-expect-error: we want to import the same version as Nextra for the main page
-import { ThemeProvider } from "next-themes"
+import { ReactElement, ReactNode } from "react"
 
-import { Footer } from "../../components/footer"
-import { NewFontsStyleTag } from "../fonts"
-import { Navbar } from "../../components/navbar/navbar"
-import { topLevelNavbarItems } from "../../components/navbar/top-level-items"
-import { MenuProvider } from "./menu-provider"
+import { Footer } from "@/components/footer"
+import { Navbar } from "@/components/navbar/navbar"
+import { topLevelNavbarItems } from "@/components/navbar/top-level-items"
 
 export default function MainLayout({
   children,
 }: {
-  children: React.ReactNode
-}) {
+  children: ReactNode
+}): ReactElement {
   return (
     <>
-      <NewFontsStyleTag />
-      <ThemeProvider attribute="class">
-        <MenuProvider>
-          <Navbar items={topLevelNavbarItems} />
-          <div className="isolate bg-neu-0 text-neu-900 antialiased">
-            {children}
-          </div>
-          <Footer />
-        </MenuProvider>
-      </ThemeProvider>
+      <Navbar items={topLevelNavbarItems} />
+      {children}
+      <Footer />
```

Comment:
Seems you lifted `.isolate` by accident.

We can try to remove `bg-neu-0 text-neu-900` classes if add them to body styles and test the old conference pages, but `isolate` was there to avoid fighting the z-index (we portal modals outside).

```suggestion
      <Navbar items={topLevelNavbarItems} />
      <div className="isolate">
        {children}
      </div>
      <Footer />
```